Tests/SonarScanner.MSBuild.Common.UnitTests/MsBuildPathSettingsTests.cs

```diff
@@ -76,7 +76,7 @@ public void GetImportBeforePaths_NonWindows_User_Account_UserProfile_Missing()
                 (Environment.SpecialFolder.UserProfile, string.Empty),
             };
 
-            var settings = new MsBuildPathSettings(GetFolderPath(paths), IsWindows(false), FileExists(true));
+            var settings = new MsBuildPathSettings(GetFolderPath(paths), IsWindows(false), DirExists(true));
```

Comment:
Minor cleanup - _FileExists_ and _DirExists_ were identical. Presumably _FileExists_ was left over from an earlier version of the code.